src/components/Dashboard/TodoList/ActionRequired.vue

```diff
@@ -7,8 +7,8 @@
           BC Registries is missing information about your business (e.g., business start date, nature of business,
           business address etc.). Please contact BC Registries to input any missing business information. Missing
           information must be entered before you can file changes or dissolve this business.
-          <v-btn class="details-btn pl-1" outlined color="primary"  :ripple="false" @click.stop="togglePanel">
-            <v-icon left color="primary" class="mr-0">mdi-information-outline</v-icon>
+          <v-btn class="details-btn" outlined color="primary" :ripple="false" @click.stop="togglePanel()">
+            <v-icon color="primary">mdi-information-outline</v-icon>
```

Comment:
This is not only cleaner but consistent with the Todo List and Filing History List layout.